content/building-blocks/store-and-document-your-data/document-data/readme-best-practices.md

```diff
@@ -121,9 +121,9 @@ The repositories below serve as examples from which you can draw inspiration for
 By default Github showcases your pinned repositories on your profile page (click on your profile picture in the top right corner > "Your profile"). A little secret is that you can add a README to your profile page by creating a new repository called `<YOUR_USERNAME>`. Make sure it's public and initialize it with a README to get started. As you can see in [this](https://www.youtube.com/watch?v=Y1z7_GfEPiE) video, you can even spice things up with emojis and gifs!
 
 {{% tip %}}
-Want to go the extra mile? Include your [GitHub Stats Card](https://github.com/anuraghazra/github-readme-stats) in your README! Simply add `https://github-readme-stats.vercel.app/api?username=<YOUR_USERNAME>` to the end of your README to incorporate a realtime widget of your number of stars, commits, PRs, issues, and contributions on Github ([see example](https://github-readme-stats.vercel.app/api?username=hannesdatta)).
+Want to go the extra mile? Include your [GitHub Stats Card](https://github.com/anuraghazra/github-readme-stats) in your README! Simply add `https://github-readme-stats.vercel.app/api?username=<YOUR_USERNAME>` to the end of your README to incorporate a real-time widget of your number of stars, commits, PRs, issues, and contributions on GitHub ([see example](https://github-readme-stats.vercel.app/api?username=hannesdatta)).
 {{% /tip %}}
 
 ## Useful Other Resources
 
-A useful external resource is [Make A Readme](https://www.makeareadme.com/) . With Make A Readme, you can create readme's for repositories. Furthermore, it provides suggestions and good tips for good Readme's. 
+A useful external resource is [Make A Readme](https://www.makeareadme.com/). With Make A Readme, you can create READMEs for repositories. Furthermore, it provides suggestions and good tips for good READMEs. 
```

Comment:
add this as a tip box instead?

Reply:
Would be good to have a "summary" box too with key takeaways?

Reply:
Got it! I've just added the mentioned segment as a tip box + added a summary section with a few takeaways at the end of the building block